.github/scripts/windows/test_task.bat

```diff
@@ -123,7 +123,7 @@ rem work-around for some spawned PHP processes requiring OpenSSL
 echo extension=php_openssl.dll >> %PHP_BUILD_DIR%\php.ini
 
 rem remove ext dlls for which tests are not supported
-for %%i in (ldap oci8_12c pdo_oci) do (
+for %%i in (ldap) do (
```

Comment:
it would be nice to test LDAP ext on windows too

Reply:
Feel free to PR that!